src/app/contexts/ToggleContext/index.test.tsx

```diff
@@ -18,6 +18,8 @@ type Props = {
 
 const TestComponent = ({ toggle, children }: PropsWithChildren<Props>) => {
   const { toggleState } = useContext(ToggleContext);
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
I'll try adding that. Its basically an annoying array indexing thing that I've never really figured out how to get around in TS. Its complaining because it doesn't know if the array can be indexed on the dynamic value, which is _fair_ but since its a test we know that it won't be an invalid value in this case.

Reply:
Changed the logic a bit to simplify it and cast the toggle index to type of `Toggles`. Still requires a `ts-expect-error` due to the indexing though, but hopefully its a little clearer at least.